explorer/app/components/Index/components/BioNetworkCell/bioNetworkCell.tsx

```diff
@@ -0,0 +1,53 @@
+import { TypographyProps } from "@databiosphere/findable-ui/lib/components/common/Typography/common/entities";
+import { processNullElements } from "app/apis/azul/common/utils";
+import * as C from "../../../../components";
+import {
+  getBioNetworkByKey,
+  getBioNetworkName,
+} from "../../../../viewModelBuilders/azul/hca-dcp/common/viewModelBuilders";
+import { NetworkIcon } from "../../../common/NetworkIcon/networkIcon";
+import { NetworkKey } from "../../common/entities";
+import { Cell } from "./bioNetworkCell.styles";
+
+const MAX_DISPLAYABLE_VALUES = 1;
+
+export interface BioNetworkCellProps {
+  label: string;
+  networkKeys: (NetworkKey | null)[];
+  TypographyProps?: TypographyProps;
+}
+
+export const BioNetworkCell = ({
+  label,
+  networkKeys,
+  TypographyProps,
+}: BioNetworkCellProps): JSX.Element => {
+  const showNTag = networkKeys?.length > MAX_DISPLAYABLE_VALUES;
+  const names = processNullElements(
+    networkKeys.map((networkKey) =>
+      networkKey === null ? null : getBioNetworkByKey(networkKey)?.key ?? null
+    )
+  );
+  return (
+    <>
+      {showNTag ? (
+        <C.NTagCell
```

Comment:
Let's import components directly inside other components e.g. change `<C.NTagCell>` to `<NTagCell>`.